_ja/functions/concat.md

```diff
@@ -0,0 +1,13 @@
+---
+name: concat
+---
+
+# `concat`
+
+@include [signatures/concat.md]
+
+`concat` は、任意の数のコレクションを受け取り、それらの要素を使って1つのコレクションを作成します。
+
+@include [figure.html source="../images/concat.svg" desc="関数 concat の図"]
+
+実際には `concat` は `Collection.concat` ではなく、具体的なコレクションオブジェクトのメソッド、例えば `List.concat` や `Set.concat` として定義されています。
```

Comment:
Not exactly the same but related to [this](https://github.com/superruzafa/visual-scala-reference/pull/11/files#r300653050).

Reply:
This paragraph was related to #5.
I will remove this added paragraph in this PR, since it requires lots of changes in many functions across translations.